hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/OzoneAddress.java

```diff
@@ -435,4 +435,19 @@ public void print(PrintStream out) {
       out.printf("Key Name : %s%n", keyName);
     }
   }
+
+  public void ensureVolumeOrBucketAddress() throws OzoneClientException {
+    if (keyName.length() > 0) {
+      if (OmUtils.isBucketSnapshotIndicator(keyName)) {
+        // If snapshot, ensure snapshot URI
+        ensureSnapshotAddress();
+      }
```

Comment:
I think we should reject key addresses here.

Reply:
Rejected key address.